runner/test/runner/pipeline_test.clj

```diff
@@ -306,6 +306,9 @@
                            statuses (->> history
                                          (map ::xt/doc)
                                          (map :status))]
+                       (is (inst? (:initialized run-info)))
+                       (is (inst? (:started run-info)))
+                       (is (inst? (:completed run-info)))
                        (is (= [:passed :running :initializing]
```

Comment:
Is there a reason why this test is not the same a the other ones `(is (contains? statuses :initializing))` , I believe would be good to have consistency here, it's a easy fix that could clear up confusion as the one I have.

Reply:
Don't quite remember honestly 😅 

Let's make it use the set+contains? as well. Probably this is some historical cruft.